scripts/setup-ubuntu.sh

```diff
@@ -1,35 +1,40 @@
 #!/bin/sh
 
-PACKAGES=""
-PACKAGES="$PACKAGES ant"			# Used by jack and aptsigner.
-PACKAGES="$PACKAGES asciidoc"
-PACKAGES="$PACKAGES automake"
-PACKAGES="$PACKAGES bison"
-PACKAGES="$PACKAGES clang"			# Used by golang, useful to have same compiler building.
-PACKAGES="$PACKAGES curl"			# Used for fetching sources.
-PACKAGES="$PACKAGES flex"
-PACKAGES="$PACKAGES gettext"			# Provides 'msgfmt' which the apt build uses.
-PACKAGES="$PACKAGES git"			# Used by the neovim build.
-PACKAGES="$PACKAGES help2man"
-PACKAGES="$PACKAGES intltool"			# Used by qalc build.
-PACKAGES="$PACKAGES libgdk-pixbuf2.0-dev"	# Provides 'gkd-pixbuf-query-loaders' which the librsvg build uses.
-PACKAGES="$PACKAGES libglib2.0-dev"		# Provides 'glib-genmarshal' which the glib build uses.
-PACKAGES="$PACKAGES libtool-bin"
-PACKAGES="$PACKAGES lzip"
-PACKAGES="$PACKAGES python3.6"
-PACKAGES="$PACKAGES subversion"			# Used by the netpbm build.
-PACKAGES="$PACKAGES tar"
-PACKAGES="$PACKAGES unzip"
-PACKAGES="$PACKAGES m4"
-PACKAGES="$PACKAGES openjdk-8-jdk"		# Used for android-sdk.
-PACKAGES="$PACKAGES pkg-config"
-PACKAGES="$PACKAGES python-docutils"		# For rst2man, used by mpv.
-PACKAGES="$PACKAGES scons"
-PACKAGES="$PACKAGES texinfo"
-PACKAGES="$PACKAGES xmlto"
-PACKAGES="$PACKAGES xutils-dev"			# Provides 'makedepend' which the openssl build uses.
+PACKAGES[0]=""
+PACKAGES[1]="ant"			# Used by jack and aptsigner.
+PACKAGES[2]="asciidoc"
+PACKAGES[3]="automake"
+PACKAGES[4]="bison"
+PACKAGES[5]="clang"			# Used by golang, useful to have same compiler building.
+PACKAGES[6]="curl"			# Used for fetching sources.
+PACKAGES[7]="flex"
+PACKAGES[8]="gettext"			# Provides 'msgfmt' which the apt build uses.
+PACKAGES[9]="git"			# Used by the neovim build.
+PACKAGES[10]="help2man"
+PACKAGES[11]="intltool"			# Used by qalc build.
+PACKAGES[12]="libgdk-pixbuf2.0-dev"	# Provides 'gkd-pixbuf-query-loaders' which the librsvg build uses.
+PACKAGES[13]="libglib2.0-dev"		# Provides 'glib-genmarshal' which the glib build uses.
+PACKAGES[14]="libtool-bin"
+PACKAGES[15]="lzip"
+PACKAGES[16]="python3.6"
+PACKAGES[17]="subversion"			# Used by the netpbm build.
+PACKAGES[18]="tar"
+PACKAGES[19]="unzip"
+PACKAGES[20]="m4"
+PACKAGES[21]="openjdk-8-jdk"		# Used for android-sdk.
+PACKAGES[22]="pkg-config"
+PACKAGES[23]="python-docutils"		# For rst2man, used by mpv.
+PACKAGES[24]="scons"
+PACKAGES[25]="texinfo"
+PACKAGES[26]="xmlto"
+PACKAGES[27]="xutils-dev"			# Provides 'makedepend' which the openssl build uses.
```

Comment:
You could also do 
PACKAGES+="ant"
PACKAGES+=" bison"
PACKAGES+=" asciidoc"
and so on, that gets rid of the for-loop below.